src/prelude/prim.mo

```diff
@@ -824,6 +824,24 @@ func getCandidLimits<system>() : {
   };
 };
 
+func setCandidTypeLimits<system>({
+  scaler : Nat32;
+  bias : Nat32;
+}) {
+  (prim "setCandidTypeLimits" : (Nat32, Nat32) -> ())(scaler, bias);
+};
```

Comment:
```suggestion
func setCandidTypeLimits<system>({
  scalar : Nat32;
  bias : Nat32;
}) {
  (prim "setCandidTypeLimits" : (Nat32, Nat32) -> ())(scalar, bias);
};
```

Reply:
Okay, that sounds like the latin name, why not.